src/app/views/public/landing/landing.controller.coffee

```diff
@@ -4,26 +4,23 @@ angular.module 'mnoEnterpriseAngular'
 
       vm = @
       vm.isLoading = true
-      vm.localProductLoading = true
       vm.highlightedApps = []
       vm.localProducts = []
 
       vm.carouselImageStyle = (app) ->
-        # Products does not have a picture
+        # Select appropriate image for carousel - logo or picture
         picture = if app.pictures then app.pictures[0] else app.logo
         {
           "background-image": "url(#{picture})"
         }
 
       MnoeMarketplace.getApps().then(
         (response) ->
-          vm.products = _.filter(response.apps, (app) -> _.includes(MnoeConfig.publicApplications(), app.nid))
```

Comment:
Is this not needed?

Reply:
As far as I can tell, there is no usages of the products list in the landing view. The products lists have been encapsulated in to the app and local product listing components.

The only thing that the landing page has direct responsibility to provide is the highlighted product list.